### PR TITLE
Issue 149: VGG-like Convolutional Neural Network Classifier

### LINKED ISSUE #149: Add VGG-like convnet primitive
Related to #121 

The architecture would be:

```
Conv2D (relu) ---> Conv2D (relu) --> MaxPooling2D --> Dropout -->
  Conv2D (relu) ---> Conv2D (relu) --> MaxPooling2D --> Dropout -->
    Flatten --> Dense(relu) --> Dropout --> Dense (softmax)
```

You can find an example of this [here](https://keras.io/getting-started/sequential-model-guide/#examples) in the **VGG-like convnet** section:
```python
import numpy as np
import keras
from keras.models import Sequential
from keras.layers import Dense, Dropout, Flatten
from keras.layers import Conv2D, MaxPooling2D
from keras.optimizers import SGD

# Generate dummy data
x_train = np.random.random((100, 100, 100, 3))
y_train = keras.utils.to_categorical(np.random.randint(10, size=(100, 1)), num_classes=10)
x_test = np.random.random((20, 100, 100, 3))
y_test = keras.utils.to_categorical(np.random.randint(10, size=(20, 1)), num_classes=10)

model = Sequential()
# input: 100x100 images with 3 channels -> (100, 100, 3) tensors.
# this applies 32 convolution filters of size 3x3 each.
model.add(Conv2D(32, (3, 3), activation='relu', input_shape=(100, 100, 3)))
model.add(Conv2D(32, (3, 3), activation='relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

model.add(Conv2D(64, (3, 3), activation='relu'))
model.add(Conv2D(64, (3, 3), activation='relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

model.add(Flatten())
model.add(Dense(256, activation='relu'))
model.add(Dropout(0.5))
model.add(Dense(10, activation='softmax'))

sgd = SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
model.compile(loss='categorical_crossentropy', optimizer=sgd)

model.fit(x_train, y_train, batch_size=32, epochs=10)
score = model.evaluate(x_test, y_test, batch_size=32)
```

### PR DESCRIPTION
**Pull request by Hector-hedb12**
Resolves #149

### CODE REVIEW

**Comment by csala** [comment]
Also @Hector-hedb12, after #143, we should try to always add a `verbose` fixed hyperparameter defaulting to `False` in all `keras.Sequential` based primitives to allow controlling their verbosity.

File: mlprimitives/jsons/keras.Sequential.VGGCNNClassifier.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+            },
+            "dropout_3_rate": {
+                "type": "float",
+                "default": 0.5,
+                "range": [
+                    0.0,
+                    0.75
+                ]
+            },
+            "batch_size": {
```
[the remark this answers is not in the document]
Comment (Hector-hedb12) [reply mlprimitives/jsons/keras.Sequential.VGGCNNClassifier.json]: 3a68063

**Comment by Hector-hedb12** [comment]
> Also @Hector-hedb12, after #143, we should try to always add a `verbose` fixed hyperparameter defaulting to `False` in all `keras.Sequential` based primitives to allow controlling their verbosity.

Added in 904a33c

File: mlprimitives/jsons/keras.Sequential.VGGCNNClassifier.json [hunk 2]
```diff
@@ -91,6 +91,14 @@
                 "type": "int",
                 "default": 20
             },
+            "verbose": {
+                "type": "int",
+                "default": 1,
```
Comment (csala) [inline mlprimitives/jsons/keras.Sequential.VGGCNNClassifier.json]: Please set the default to 0

Comment (Hector-hedb12) [reply mlprimitives/jsons/keras.Sequential.VGGCNNClassifier.json]: 24c9abd

**Comment by csala** [comment]
Thanks @Hector-hedb12 !

**Pull request merged**